From d9d8bdcde6ec14d0db4bf2d0cff1ecf1b414f370 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 15 Jan 2018 15:34:13 +0800
Subject: [PATCH 3198/5242] MLK-17370 video: fbdev: mxc_edid: change '640x480'
 mode clock

commit  270b89e35fc23bf5a20d05bc1c5a89bf66a2bc5a from
https://source.codeaurora.org/external/imx/linux-imx.git

For the '640x480' cea mode, change the 'pixclock' from
39722 ps to 39683 ps to satisfy the imx7ulp hdmi display
requirement.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/mxc_edid.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/mxc/mxc_edid.c b/drivers/video/fbdev/mxc/mxc_edid.c
index 23110ce..02563d8 100644
--- a/drivers/video/fbdev/mxc/mxc_edid.c
+++ b/drivers/video/fbdev/mxc/mxc_edid.c
@@ -41,7 +41,7 @@
 const struct fb_videomode mxc_cea_mode[64] = {
 	/* #1: 640x480p@59.94/60Hz 4:3 */
 	[1] = {
-		NULL, 60, 640, 480, 39722, 48, 16, 33, 10, 96, 2, 0,
+		NULL, 60, 640, 480, 39683, 48, 16, 33, 10, 96, 2, 0,
 		FB_VMODE_NONINTERLACED | FB_VMODE_ASPECT_4_3, 0,
 	},
 	/* #2: 720x480p@59.94/60Hz 4:3 */
-- 
1.7.9.5

